saleor/checkout/calculations.py

```diff
@@ -254,6 +257,12 @@ def fetch_checkout_prices_if_expired(
     if tax_data:
         _apply_tax_data_from_app(checkout, lines, tax_data)
 
+    if site_settings is None:
+        site_settings = Site.objects.get_current().settings
+
+    if checkout.tax_exemption and site_settings.include_taxes_in_prices:
```

Comment:
Same as in the order part: https://github.com/saleor/saleor/pull/10344/files#r962785718